README.md

```diff
@@ -33,6 +33,25 @@ production-worthy implementation in 2021.
 Communications for Rust-Lightning and Lightning Development Kit happens through
 [LDK slack](http://lightningdevkit.org/).
 
+Crates
+-----------
+1. [lightning](./lightning)   
```

Comment:
nit: several lines have trailing whitespace, including this one.

Reply:
It was only for one line AFAIK. I fixed it. Let me know if there are other\.